src/iexamine.cpp

```diff
@@ -2780,6 +2783,19 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
+{
+    item milk( "milk", 0, 1 );
+    if( !source_mon->has_effect( effect_milked ) ) {
```

Comment:
Decided to combine those two if in one as I haven't realized that I could do that instead of making 2 if for each particular case.